docs/pim/attributes/date_and_time.md

```diff
@@ -0,0 +1,77 @@
+---
+description: Date and time attribute type allows you to store product information related to time, like an expiration date or date of manufacturing.
+---
+
+# Date and time attribute
+
+The date and time attribute type is one of the [attribute types](products.md#attributes) that allows you to efficiently represent date and time values as part of the product specification in the [Product Information Management](pim_guide.md) system.
```

Comment:
```suggestion
The date and time [attribute type](products.md#attributes) allows you to efficiently represent date and time values as part of the product specification in the [Product Information Management](pim_guide.md) system.
```
Can you "efficiently" represent time and date? Why not just "represent"?

Reply:
`Represent` is good, thanks :D Will be removed in a separate commit.